citar.el

```diff
@@ -99,21 +98,46 @@ to include."
   :type '(repeat string))
 
 (defcustom citar-templates
-  '((main . "${author editor:30}     ${date year issued:4}     ${title:48}")
-    (suffix . "          ${=key= id:15}    ${=type=:12}    ${tags keywords keywords:*}")
-    (preview . "${author editor} (${year issued date}) ${title}, ${journal journaltitle publisher container-title collection-title}.\n")
-    (note . "Notes on ${author editor}, ${title}"))
+  '((main . "%->30a     %4d     %->48t")
+    (suffix . "          %>15k    %>12T   %K")
+    (preview . "%>a (%d) %t, %p.\n")
```

Comment:
What should the date syntax be here?

Reply:
Should be `%>4d` also, to restrict it to the year.

Reply:
So IUC, still:

- [ ] this
- [ ] the truncation issue above
- [ ] docs update

Reply:
Yes. I'll commit this template fix with the truncation solution I suggested above (adding `(full-fields (citar--format-entry entry "%a %t")` to `citar-format-candidates`.

Working on the docs now.

Reply:
> Wondering if there should be some kind of "breaking change" notification up top, for anyone who has customized `citar-templates`?

Yeah, I was wondering about that too. We do have a changelog, but never update it!

Reply:
Actually, can you please just add the "Breaking Changes" section towards the top?

Ideally we also get back to updating the changelog, where we can put more fine-grained details, but to avoid needless panic and frustration, would be good to add this to the README.

Reply:
Maybe the **!!! BREAKING CHANGE NOTICE !!!** is overdoing it a bit? 🤷‍♂️